ci/appveyor/setup.ps1

```diff
@@ -0,0 +1,8 @@
+$env:PATH="$env:PYTHON;$env:PYTHON\\Scripts;$env:PATH"
+
+Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.msi"
+Start-Process -FilePath "msiexec.exe" -ArgumentList "/i rust-nightly-$env:TARGET.msi INSTALLDIR=`"$((Get-Location).Path)\rust-nightly-$env:TARGET`" /quiet /qn /norestart" -Wait
+$env:PATH="$env:PATH;$((Get-Location).Path)/rust-nightly-$env:TARGET/bin"
+
+$pythonLocation = Invoke-Expression "python -c `"import sys; print(sys.base_prefix)`""
+$env:LIBPATH = "$env:LIBPATH; $( Join-Path $pythonLocation "libs" )"

```

Comment:
it is similar to this line
https://github.com/PyO3/pyo3/blob/c4c75bbf811773414d31ae9205988f47ee876399/ci/travis/setup.sh#L35